rye/src/cli/publish.rs

```diff
@@ -111,8 +119,8 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
         .arg("--no-color")
         .arg("upload")
         .args(files)
-        .arg("--user")
```

Comment:
How did this even work? Pretty sure twine does not accept `--user`?

Reply:
from https://twine.readthedocs.io/en/stable/#twine-upload

```
  -u USERNAME, --username USERNAME
                        The username to authenticate to the repository (package index)
                        as. (Can also be set via TWINE_USERNAME environment variable.)
```

This may be an accidental mistake.